packages/qemu-system-x86-64-headless/build.sh

```diff
@@ -78,7 +82,7 @@ termux_step_configure() {
 		--enable-virtfs \
 		--enable-curl \
 		--enable-fdt \
-		--disable-kvm \
+		--enable-kvm \
```

Comment:
Do you have a proof that KVM works on modern Android devices? Last time I have tried it, on Samsung Galaxy S7 with custom kernel, it didn't worked. Reason: hyp (EL2) mode is unavailable, which is somewhat expected since devices with ARM TrustZone already use hypervizor for own purposes.

Reply:
You are right, KVM doesn't work on most devices
But i have recompiled kernel for realme 6i and i had /dev/kvm file and Phone was booted in EL2
At that time, i wanted to test kvm acceleration and install qemu on termux but later i realised that kvm support is disabled
So i did recompile qemu with kvm support.
I thought this is actually time and data wasting to recompile qemu.
That's the reason why i enabled kvm support

Reply:
> But i have recompiled kernel for realme 6i and i had /dev/kvm file and Phone was booted in EL2

Ok, thanks.